.github/workflows/cd.yml

```diff
@@ -0,0 +1,88 @@
+name: Continuous Deployment
+
+on:
+  release:
+    types: [created]
+
+env:
+  CARGO_TERM_COLOR: always
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  publish-artifacts:
+    name: Generating artifacts for ${{ matrix.job.target }}
+    runs-on: ${{ matrix.job.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        job:
+          - os: macos-latest
+            target: x86_64-apple-darwin
+          - os: ubuntu-20.04
```

Comment:
Is there any reason to use `20.04` instead of `latest`? Probably doesn't matter but just curious

Reply:
Ahh okay that rings a bell haha, thanks

Reply:
As I did nor recall this, I was just gonna try with ubuntu latest